config/crds/cluster_v1alpha1_machine.yaml

```diff
@@ -83,7 +83,7 @@ spec:
               properties:
                 controlPlane:
                   description: Semantic version of the Kubernetes control plane to
-                    run. This should only be populated when the machine is a master.
+                    run. This should only be populated when the machine is a controlplane.
```

Comment:
If these aren't auto-generated, then they should all read:

"when the machine is intended to host the cluster's control plane."